global/concourse/README.md

```diff
@@ -1,246 +0,0 @@
-# Concourse Helm Chart
```

Comment:
It would be nice to have a basic README explaining what this chart is how we did it.

Reply:
Eat least we should briefly explain the following things:
* Daemonset per team with
* Logging reverse proxy